cmd/maestro/agent/cmd.go

```diff
@@ -70,7 +70,6 @@ func runAgent(cmd *cobra.Command, args []string) {
 	cfg := spoke.NewWorkAgentConfig(commonOptions, agentOption)
 	cmdConfig := commonOptions.CommoOpts.
 		NewControllerCommandConfig("maestro-agent", version.Get(), cfg.RunWorkloadAgent)
-	cmdConfig.DisableLeaderElection = true
```

Comment:
@clyang82 I found this was added in https://github.com/openshift-online/maestro/pull/20, any reason to disable leader election explicitly?

Reply:
no specific reason to disable it at the beginning. just make it works for initial version. We need to enable it based on the current case.

Reply:
Updated this to following https://github.com/open-cluster-management-io/ocm/blob/main/pkg/cmd/spoke/work.go#L24

One question: we have add some flag alias for maestro agent, which is different from work-agent. Eg. `--consumer-name` instead of `--spoke-cluster-name`? do we still need these alias flags?

Reply:
what does election leader provide that advisory locks from postgres do not?

locks give us concurrency and parallelism in execution.  leader election ties us to a single pod at a time and has proven troublesome with CS in the past.

Reply:
ah, i think this is in the context of the agent, which would be in a cluster _without_ the postgres database.  nevermind my question.

Reply:
yes, the agent is using in-cluster leader election.